TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot_components/robot/Robot.java

```diff
@@ -86,16 +126,16 @@ public void dropGameElement() {
     //manages up/down positions of intake
     public void toggleBucket(){
```

Comment:
this should get removed because we aren't using a servo on the intake anymore

Reply:
done